lib/views/widgets/selected_field.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class SelectedInput extends StatefulWidget {
+  final List<String> items;
+  String initialValue;
+
+  SelectedInput({
+    Key? key,
+    required this.items,
+    required this.initialValue,
+  }) : super(key: key);
+
+  @override
+  State<SelectedInput> createState() => _SelectedInputState();
+}
+
+bool Select = false;
```

Comment:
move this line inside _SelectedInputState and use loweCamelCase when naming ([reference](https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html))

suggestion for name: isSelected